.travis.yml

```diff
@@ -9,14 +9,14 @@ env:
   global:
     - ROS_REPO=ros
   matrix:
-    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=file
-    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=debian
```

Comment:
can this repo be made 3-distro-compatible?
does the melodic migration break indigo?

Reply:
It should still work in indigo. But I removed indigo since it is end-of-life and to speed up the CI tests.

Reply:
Please undo the removal of indigo_dev. If the branch name is indigo, we should at least ensure that it still works with indigo.

Furthermore, I will need to check if the removed lines in libconcorde_tsp_solver/CMakeLists.txt will not break the functionality. Otherwise, we might need to create a new melodic branch.

Reply:
Ok, I re-added indigo.
In the long term, the CI pass should not depend on indigo. Users are advised to switch to newer Ubuntu/ROS versions.

Reply:
Thanks, you are right, but we have some legacy client installations that still need the working indigo_dev version. I am totally fine with starting a new kinetic_dev or melodic_dev branch for future use, but we should not break the indigo_dev branch. Do you agree @ipa-jba @fmessmer ?